src/Inventory.zig

```diff
@@ -630,6 +674,49 @@ pub const Command = struct { // MARK: Command
 					}};
 
 					return out;
+				},
+				.health => {
+					if (data.len != 8) {
+						return error.Invalid;
+					}
+
+					const id = std.mem.readInt(u32, data[0..4], .big);
+
+					var target: *main.server.User = undefined;
+					const userList = main.server.getUserListAndIncreaseRefCount(main.stackAllocator);
+					defer main.server.freeUserListAndDecreaseRefCount(main.stackAllocator, userList);
+					for (userList) |user| {
+						if (user.id == id) {
+							target = user;
+							break;
+						}
+					}
+
+					return .{.health = .{
+						.target = target,
+						.health = @bitCast(std.mem.readInt(u32, data[4..8], .big))
+					}};
+				},
+				.kill => {
+					if (data.len != 4) {
+						return error.Invalid;
+					}
+
+					const id = std.mem.readInt(u32, data[0..4], .big);
+
+					var target: *main.server.User = undefined;
+					const userList = main.server.getUserListAndIncreaseRefCount(main.stackAllocator);
```

Comment:
This code is executed on the client side. You must not assume that there is a valid server instance running.